cassandra-catalog.bom

```diff
@@ -63,12 +63,13 @@ brooklyn.catalog:
 
       brooklyn.config:
         cassandra.archive.url: http://archive.apache.org/dist/cassandra/3.5/apache-cassandra-3.5-bin.tar.gz
+        cql.port: 9042
+        gossip.port: 7000
         provisioning.properties:
           minRam: 4g
 
       shell.env:
         C_URL: $brooklyn:config("cassandra.archive.url")
-        C_CLUSTER_NAME: $brooklyn:config("cassandra.cluster.name")
```

Comment:
I believe this deletion was done unintentionally, as the cluster name is one of the few config keys that we support.